crates/bsp_types/src/extensions/rust/rust_toolchain.rs

```diff
@@ -29,7 +29,8 @@ pub struct RustToolchainResult {
 #[derive(Serialize, Deserialize, Default, PartialOrd, PartialEq, Ord, Eq)]
 #[serde(rename_all = "camelCase")]
 pub struct RustToolchainItem {
-    /** Additional information about Rust toolchain. */
+    /** Additional information about Rust toolchain.
+    Obtained from `rustc`. */
```

Comment:
czy to przechodzi formatowanie? chyba domyślnie wcięcie powinno być takie
```suggestion
        Obtained from `rustc`. */
```

Reply:
Tak mieliśmy do tej pory w całym `bsp-types`. Kasia też tak to będzie generować i przechodzi formatowanie bez problemu

Reply:
spoko